src/views/evaluations/evaluations.js

```diff
@@ -0,0 +1,108 @@
+function EvaluationsPage(options) {
+    const newOptions = options || {};
+    const headings = [
+        'Nume', 'Tehnologie', 'Nivel', ''
+    ];
+    const rows = [
+        {
+        name: 'Popescu Adrian',
+        technology: 'Javascript',
+        level: 'Mid 2'
+    },
+    {
+        name: 'Dragan Roxana',
+        technology: 'PHP',
+        level: 'Mid 1'
+    },
+    {
+        name: 'Florescu Mihai',
+        technology: 'Javascript',
+        level: 'Junior 3'
+    },
+    {
+        name: 'Ghoreghe Andrei',
+        technology: 'Ruby',
+        level: 'Mid 2'
+    }
+];
+    return `
+    ]
+    ${NAV()}
+    ${EvaluationsTable({
+        items: rows,
+        itemHeadings: headings
+    })}
+    ${Footer()}
+    `
+    ;
+}
+    
+function EvaluationsTableHeader(options) {
+        const headings = [];
+
+        options.headings.forEach(function(el) {
+            const generatedEl = `
+            <th>${el}</th>`;
+            headings.push(generatedEl);
+        });
+
+        const headingsEl = `
```

Comment:
This can just be `const headingsEl = headings.join('');` to keep things nice and simple.